android/assets/jsons/translations/template.properties

```diff
@@ -66,6 +66,8 @@ Sounds good! =
 Not this time. = 
 Excellent! = 
 How about something else... = 
+#Owned resources
+Owned: = 
```

Comment:
`Owned: [amountOwned] = `
Already present in template, just not used

Reply:
line 43 in [TradePopup.kt](https://github.com/yairm210/Unciv/blob/master/core/src/com/unciv/ui/worldscreen/TradePopup.kt)
should be `tradeText += "\n" + "Owned: [${ourResources[offer.name]}]".tr()`